QuickStart/ViewController.swift

```diff
@@ -0,0 +1,276 @@
+    
+    var applicationContext : ADAuthenticationContext?
+    var accessToken = String()
+    
+    @IBOutlet weak var loggingText: UITextView!
+    @IBOutlet weak var signoutButton: UIButton!
+
+
+    override func viewDidLoad() {
+        
+        /**
+         Initialize a ADAuthenticationContext with a given authority
+         
+         - authority:           A URL indicating a directory that ADAL can use to obtain tokens. In Azure AD
+                                it is of the form https://<instance/<tenant>, where <instance> is the
+                                directory host (e.g. https://login.microsoftonline.com) and <tenant> is a
+                                identifier within the directory itself (e.g. a domain associated to the
+                                tenant, such as contoso.onmicrosoft.com, or the GUID representing the
+                                TenantID property of the directory)
+         - error                The error that occurred creating the application object, if any, if you're
+                                not interested in the specific error pass in nil.
+         */
+        
+        self.applicationContext = ADAuthenticationContext(authority: kAuthority, error: nil)
+        super.viewDidLoad()
+
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+
+        super.viewWillAppear(animated)
+        signoutButton.isEnabled = (currentAccount()?.accessToken != nil)
+    }
+    
+    /**
+     This button will invoke the authorization flow.
+    */
+
+    @IBAction func callGraphButton(_ sender: UIButton) {
+
+        if self.currentAccount() == nil {
+            // We check to see if we have a current logged in account.
+            // If we don't, then we need to sign someone in.
+            self.acquireTokenInteractively()
+        } else {
+            self.acquireTokenSilently()
+        }
+    }
+
+    func acquireTokenInteractively() {
+        
+        guard let applicationContext = self.applicationContext else { return }
+
+        /**
+         
+         Acquire a token for an account
+         
+         - withResource:        The resource you wish to access. This will the Microsoft Graph API for this sample.
+         - clientId:            The clientID of your application, you should get this from the app portal.
+         - redirectUri:         The redirect URI that your application will listen for to get a response of the Auth code after authentication. Since       this a native application where authentication happens inside the app itself, we can listen on a custom URI that the SDK knows to look for from within the application process doing authentication.
+         - completionBlock:     The completion block that will be called when the authentication
+         flow completes, or encounters an error.
+         */
+
+        applicationContext.acquireToken(withResource: kGraphURI, clientId: kClientID, redirectUri:kRedirectUri){ (result) in
```

Comment:
I'd rather keep `acquireToken()` since it's the default. The fact I do the logic of `acquireToken()` is weird anyway. I'll ask you on separate thread about possible refactor.